### ISSUE TITLE
[Instrumentation] Agent Stuck

### ISSUE THREAD

**Issue by llmhyy** [llmhyy/microbat#119]
hi @lylytran 

The agent stucks given the following cmd, please kindly have a check (Mockito-14), many thanks!

```
E:\linyun\bug_repo\jdk1.7.0_800\bin\java -Xmx30g -ea -noverify -javaagent:E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\instrumentator.jar=precheck=false,excludes=java.util.Properties*,java_home=E:\linyun\bug_repo\jdk1.7.0_800,stepLimit=100000,log=printProgress;info,working_dir=E:\linyun\bug_repo\Mockito\14\bug,varLayer=1,includes=java.lang.reflect.AccessibleObject;java.lang.reflect.Field;java.lang.Class;java.lang.reflect.Method;java.lang.AbstractStringBuilder;java.lang.StringBuilder;java.lang.StackTraceElement;java.util.List;java.util.Stack;java.util.ArrayList;java.util.Collections$UnmodifiableList;java.util.Vector;java.util.Collections$EmptyList;java.util.Collections$UnmodifiableRandomAccessList;java.util.AbstractSequentialList;java.util.LinkedList;java.util.AbstractList;java.util.Collections$SynchronizedList;java.util.RandomAccessSubList;java.util.SubList;java.util.AbstractCollection;java.util.Arrays;java.lang.Thread;java.lang.ThreadLocal;org.mockito.cglib.proxy.Factory;java.util.Map;java.lang.ClassValue$ClassValueMap;java.util.LinkedHashMap;sun.nio.cs.StandardCharsets$Aliases;java.util.concurrent.ConcurrentHashMap;java.util.HashMap;sun.util.PreHashedMap;java.io.ExpiringCache$1;java.util.Properties;java.util.AbstractMap;sun.nio.cs.StandardCharsets$Cache;java.util.jar.Attributes;java.util.WeakHashMap;java.util.concurrent.ConcurrentMap;java.util.Hashtable;java.util.Collections$EmptyMap;sun.nio.cs.StandardCharsets$Classes;java.util.Collections$SingletonMap;java.util.SortedMap;java.util.TreeMap;java.util.NavigableMap;java.util.Iterator;java.util.HashMap$HashIterator;java.util.Hashtable$Enumerator;java.util.HashMap$EntryIterator;java.util.ArrayList$Itr;java.util.ServiceLoader$LazyIterator;java.util.ServiceLoader$1;java.util.LinkedHashMap$EntryIterator;java.util.LinkedHashMap$LinkedHashIterator;java.util.Collections$EmptyIterator;java.util.HashMap$ValueIterator;java.util.LinkedHashMap$ValueIterator;java.util.ArrayList$ListItr;java.util.LinkedList$ListItr;java.util.SubList$1;java.util.AbstractList$ListItr;java.util.IdentityHashMap$KeyIterator;java.util.IdentityHashMap$IdentityHashMapIterator;java.util.Collections,launch_class=org.mockitousage.bugs.VerifyingWithAnExtraCallToADifferentMockTest,class_path=E:\linyun\bug_repo\Mockito\14\bug\target\test-classes;E:\linyun\bug_repo\Mockito\14\bug\target\classes;E:\linyun\bug_repo\Mockito\14\bug\lib\build\asm-3.1.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\bnd-0.0.313.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\jarjar-1.0.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\jaxen-1.1.1.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\maven-ant-tasks-2.0.9.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\pmd-4.1.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\sorcerer.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\compile\com.springsource.org.junit-4.5.0.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\repackaged\cglib-and-asm-1.0.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\run\com.springsource.org.hamcrest.core-1.1.0.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\run\com.springsource.org.objenesis-1.0.0.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\sources\cglib-and-asm-1.0-sources.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\sources\com.springsource.org.hamcrest.core-1.1.0-sources.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\sources\com.springsource.org.objenesis-1.0.0-sources.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\test\fest-assert-1.1.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\test\fest-util-1.1.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\test\powermock-reflect-1.2.5.jar;E:\linyun\bug_repo\Mockito\lib\objenesis-1.2.jar;E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\junit.jar;E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\org.hamcrest.core.jar;E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\testrunner.jar;E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\bcel-6.0.jar;E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\javassist.jar,dump_file_path=E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\trace\Mockito\14\bug.exec,expectedSteps=4568 -cp E:\linyun\bug_repo\Mockito\14\bug\target\test-classes;E:\linyun\bug_repo\Mockito\14\bug\target\classes;E:\linyun\bug_repo\Mockito\14\bug\lib\build\asm-3.1.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\bnd-0.0.313.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\jarjar-1.0.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\jaxen-1.1.1.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\maven-ant-tasks-2.0.9.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\pmd-4.1.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\build\sorcerer.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\compile\com.springsource.org.junit-4.5.0.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\repackaged\cglib-and-asm-1.0.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\run\com.springsource.org.hamcrest.core-1.1.0.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\run\com.springsource.org.objenesis-1.0.0.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\sources\cglib-and-asm-1.0-sources.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\sources\com.springsource.org.hamcrest.core-1.1.0-sources.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\sources\com.springsource.org.objenesis-1.0.0-sources.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\test\fest-assert-1.1.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\test\fest-util-1.1.jar;E:\linyun\bug_repo\Mockito\14\bug\lib\test\powermock-reflect-1.2.5.jar;E:\linyun\bug_repo\Mockito\lib\objenesis-1.2.jar;E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\junit.jar;E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\org.hamcrest.core.jar;E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\testrunner.jar;E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\bcel-6.0.jar;E:\linyun\software\eclipse-java-mars\eclipse-java-mars-clean\eclipse\dropins\junit_lib\javassist.jar microbat.evaluation.junit.MicroBatTestRunner org.mockitousage.bugs.VerifyingWithAnExtraCallToADifferentMockTest shouldAllowVerifyingWhenOtherMockCallIsInTheSameLine
```

**Comment by llmhyy** [comment]
probably, it just takes a very long time to store the trace. Anyway, it's better to have a check for this.

**Comment by lylytran** [comment]
fixed at commit 5e64ddb714eada56835a1470506d39cf153c7f11 [5e64ddb]

**Comment by llmhyy** [comment]
done
